### ISSUE TITLE
rule: padded-blocks: false positive with comments

### ISSUE THREAD

**Issue by joeybaker** [eslint/eslint#1938]
With `padded-blocks` set to `"never"` the following is reported as a warning and should not be:

```js
if (a){
  // i'm a comment!
 let b = c
}
```

Removing the comment allows the rule to pass.

**Comment by gyandeeps** [comment]
I think it doesn't consider comments. Possibly a bug.

**Comment by gyandeeps** [comment]
Can you paste the error message with line number. Because on my end I dont get any errors.

**Comment by joeybaker** [comment]
Sure: 

```
22:31  warning  Block must not be padded by blank lines  padded-blocks
```

Comes from
```js
rebundle = function rebundle() {
  // a thing
```

**Comment by gyandeeps** [comment]
Can you share more code as the above doesnt help.  
Are you sure there are no other characters there. Still above scenarios works fine on my side.

**Comment by joeybaker** [comment]
doh! My bad. This is actually coming from `babel-eslint`. Sorry for wasting your time.
